test/propagation.jl

```diff
@@ -108,12 +108,12 @@ using LinearAlgebra: norm
 
     etv = range(sol64.t0, sol64.t[end], 5)
     for et in eachindex(etv)
```

Comment:
I'm just realizing this should read (with the suggestion above):

```julia
for t in tv
```
 and corresponding changes below...

Reply:
Done.